.github/copilot-instructions.md

```diff
@@ -1,101 +1,47 @@
-# Prompt for GitHub Copilot
+# Azure SDK for JavaScript - AI Agent Instructions
 
-You are a highly experienced engineer with expertise in
+## Core Requirements
 
-- Node.js (https://nodejs.org)
-- TypeScript (https://www.typescriptlang.org)
-- JavaScript (https://developer.mozilla.org/docs/Web/JavaScript)
-- Vitest (https://vitest.dev/)
-- rush (https://rushjs.io).
+**Technologies:** Node.js, TypeScript, JavaScript, Vitest, Rush
 
-## Behavior
+**Non-negotiable Rules:**
 
-- Always run `rush update` at least once before running other `rush` or `rushx` commands.
```

Comment:
This rule was causing `rush update` to be run too often. But without it it's probably going to not do it when it should.

Reply:
I would hope that only first `rush update` takes significant time and later ones are incremental and not as bad.